### ISSUE TITLE
Any way to use the floating button via layout XML instead of via code?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [oguzbilgener/CircularFloatingActionMenu#19]
I don't really want it to always be there.
For example, if there is a navigation drawer, it shouldn't be visible, and others might be shown (when choosing other fragments to show).
I think it's better to let us choose exactly where to put it on the layout file.

**Comment by onewilk** [comment]
Good point,i don't want to write so many java code too...and you miss the onClickListener and onItemClickListener ...
